release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2411,6 +2434,17 @@ In the following tables, features are marked with the following statuses:
 [cols="4,1,1,1",options="header"]
 |====
 |Feature |4.12 |4.13 |4.14
+
+|External load balancers with installer-provisioned infrastructure
```

Comment:
Just double-checking - is this correct? I though we've added that in a previous release?

Reply:
It's correct, external load-balancers is GA in 4.14